apps/frontend/src/components/ViewFormsList/CSVImport.tsx

```diff
@@ -0,0 +1,65 @@
+import { Box, Text, Button } from '@chakra-ui/react';
+import { useRef } from 'react';
+import { putMultipleCSVForms } from '../../utils/sendRequest';
+
+/**
+ * A component that allows the user to upload a CSV file from their
+ * computer to import completed forms into the database. Includes a
+ * text description and the Import button.
+ */
+export default function CSVImport() {
+  const inputFile = useRef<HTMLInputElement>(null);
+
+  // called after the user uploads a file
+  // sends the uploaded file to the endpoint if it is a CSV file, or
+  // alerts the user if they uploaded the wrong file type
+  async function handleFileUpload() {
+    const file_input = inputFile.current!;
```

Comment:
This feels dangerous to me. What happens here if inputFile is null? I'm not entirely sure but I think it would throw an error. Also the naming here of `inputFile` and `file_input` isn't super helpful.

Lets combine lines 17 and 18 into `const files = inputFile.current?.files`, and then change the if statement to be `if (files && files.length === 1)`. 
Then, line 27 can be `inputFile.current.value = ''` and can be moved inside the if statement so that current is defined.